Sources/Fuzzilli/Core/JavaScriptEnvironment.swift

```diff
@@ -195,20 +195,27 @@ public class JavaScriptEnvironment: ComponentBase, Environment {
         logger.info("Have \(customPropertyNames.count) custom property names: \(customPropertyNames)")
         logger.info("Have \(customMethodNames.count) custom method names: \(customMethodNames)")
     }
-    
+
+    public func constructables() -> [String] {
```

Comment:
I think this should be a property and not a method, so that the content is computed just once.